scripts/check_compliance.py

```diff
@@ -947,7 +947,7 @@ def set_commit_status(status, msg):
                              "pylint"}:
                 msg = "```\n" + msg + "\n```"
             fail_msg += msg + "\n\n"
-            set_commit_status('failure', 'Checks failed')
+            set_commit_status('failure', 'Checks failed on run ' + build_number)
```

Comment:
maybe `set_commit_status()` could be updated instead:

```python
    def set_commit_status(status, msg):
        # 'case' gets set in the loop.
        # pylint: disable=undefined-loop-variable
        github_commit.create_status(
            status, name2doc[case.name],
            "{} ({})".format(msg, build_number), case.name)
```

Then the build number should get included for all types of statuses.

Reply:
DOne.